rules/naming/src/PropertyRenamer.php

```diff
@@ -40,15 +40,17 @@ public function __construct(
         $this->nodeNameResolver = $nodeNameResolver;
     }
 
-    public function rename(PropertyRename $propertyRename): void
+    public function rename(PropertyRename $propertyRename): bool
     {
         if ($this->breakingVariableRenameGuard->shouldSkipProperty($propertyRename)) {
-            return;
+            return false;
         }
 
         $onlyPropertyProperty = $propertyRename->getProperty()->props[0];
         $onlyPropertyProperty->name = new VarLikeIdentifier($propertyRename->getExpectedName());
         $this->renamePropertyFetchesInClass($propertyRename);
+
+        return true;
```

Comment:
Return book on non bool method cannot be seen from name. Any idea how to decouple this?

Reply:
I can't say I like this approach but check this commit out 61cb7e4b6483c3acdc672542adf78807abce55e4

Reply:
That create space for bugs, since the value is set once `true` for all following cases.

This part is tricky. I wonder if the `RenamePropertyToMatchTypeRector` rule could work with subscribe to `Property` and `PropertyFetch` instead.

Reply:
At the beginning of the method, there is `$this->isRenamed = false` to prevent this kind of bugs but anyway, some sort of bugs could appear.